examples/gitserver/hooks/buildconfig-autostart/post-receive

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+commits=$(mktemp)
+cat > ${commits}
+
+function startbuild { 
+  bcref=$1
+  bcref=(${bcref//./ })
+  bc=${bcref[0]}
+  ref=${bcref[1]:-master}
+  
+  if webhook=$(oc start-build --list-webhooks="generic" "${bc}" | head -n 1); then
```

Comment:
i don't get why you care if the buildconfig has a webhook or why you're using the webhook to start the build instead of just oc start-build?